src/utils/encryption.ts

```diff
@@ -0,0 +1,18 @@
+import { ModeOfOperation, utils } from 'aes-js';
+import SHA256 from 'sha256';
+
+export const decrypt = (data: string, key: string) => {
+  const aes = new ModeOfOperation.ctr(utils.hex.toBytes(key));
+  const decryptedBytes = aes.decrypt(utils.hex.toBytes(data));
+  return utils.utf8.fromBytes(decryptedBytes);
+};
+
+export const encrypt = (data: string, key: string) => {
+  const aes = new ModeOfOperation.ctr(utils.hex.toBytes(key));
+  const encryptedBytes = aes.encrypt(utils.utf8.toBytes(data));
+  return utils.hex.fromBytes(encryptedBytes);
+};
+
+export const hashPassword = (password: string) => {
+  return utils.hex.fromBytes(SHA256(password, { asBytes: true }));
```

Comment:
SHA256 hashes are easy to compute, potentially making bruteforcing easier, so it shouldn't be used to derive a cryptographic key. Consider using something like [PBKDF2](https://en.wikipedia.org/wiki/PBKDF2) with a high number of iterations.

Reply:
We use SHA256 on beta, which is why I initially just used it. I've reached out to Harry for a recommendation on what to use.